### PR TITLE
created bookings routes and controller

### PR DESCRIPTION
**Pull request by JoanaFCruz**
review pls

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+  end
+
+  def new
+  end
+
+  def create
+
+  end
+
```
Comment (FlaviaAssumpcao) [inline app/controllers/bookings_controller.rb:18]: Add destroy method.

Can leave edit out to simplify if we want.

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+class BookingsController < ApplicationController
+  before_action :set_experience, only: [:new, :create]
+  before_action :set_booking, only: [:show]
```
Comment (FlaviaAssumpcao) [inline app/controllers/bookings_controller.rb]: will need to add destroy

File: app/controllers/bookings_controller.rb (hunk 1 above)
Comment (JoanaFCruz) [reply app/controllers/bookings_controller.rb:18]: added. Can you check?

File: config/routes.rb [hunk 3]
```diff
@@ -5,5 +5,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-  resources :experiences
+  resources :experiences do
+    resources :bookings, only: [:new, :create]
+  end
+  resources :bookings, only: [:index, :show]
```
Comment (FlaviaAssumpcao) [inline config/routes.rb]: add destroy

**Pull request merged**